src/main/java/org/sbx/objects/Door.java

```diff
@@ -0,0 +1,18 @@
+package org.sbx.objects;
+
+import org.sbx.interfaces.Site;
+
+/**
+ * Created by aloginov on 27.09.16.
+ */
+public class Door implements Site {
+
+    private Room room1;
```

Comment:
I don't think that a door should have rooms inside. Think from a model perspective. What should be relations?